### PR TITLE
ft: add multiple site setup script

### PR DESCRIPTION
**Pull request by philipyoo**
StorageClass uses a comma-separated list of target site names.
Role Arn is just the source role Arn or if there is a destination zenko site, role arn is a comma separated source role Arn and target Zenko role Arn

### CODE REVIEW

File: bin/replication.js [hunk 1]
```diff
@@ -105,6 +105,49 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-setup')
+    .description('NOTE: Only works for external sites')
+    .option('--source-bucket <name>', '[required] source bucket name')
+    .option('--source-profile <name>',
+            '[required] source aws/credentials profile')
+    .option('--target-bucket <name>', '[required] target bucket names')
```
Comment (bennettbuchanan) [inline bin/replication.js]: Maybe make this optional in case the target is a Scality destination.

File: bin/replication.js [hunk 2]
```diff
@@ -105,6 +105,49 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-setup')
+    .description('NOTE: Only works for external sites')
```
Comment (bennettbuchanan) [inline bin/replication.js]: Why does it only work for external sites?

**Review by bennettbuchanan — DISMISSED** [review]
Thanks for this. Left a couple things to consider but otherwise LGTM! :shipit:

File: bin/replication.js [hunk 3]
```diff
@@ -105,6 +105,49 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-setup')
```
Comment (bennettbuchanan) [inline bin/replication.js]: Perhaps 'multi-site-setup' would be more descriptive.

File: bin/replication.js (hunk 2 above)
Comment (philipyoo) [reply bin/replication.js]: I wasn't sure how to include another Scality site in a one-to-many configuration.
If I were to include a Scality site, would I go through the same process of a one-to-one setup to another Scality site, setting the arn to `sourceArn,targetArn` but also setting StorageClass to the target external site names (excluding the site name for the target Scality site)?

Comment (bennettbuchanan) [reply bin/replication.js]: Exactly! You would include the Scality replication endpoint name as one of the comma-separated storage classes. Then the role would be a comma-separated pair of role ARNs (i.e., `sourceArn,targetArn`) and the destination bucket ARN would be the bucket in that Scality instance to replicate to.

File: bin/replication.js [hunk 4]
```diff
@@ -105,6 +110,73 @@ program.version(version);
...
+        const targetProfile = options.targetProfile;
+        const multiSites = options.multiSites;
+
+        const destinationEndpoint = config.getBootstrapList()
+            .find(dest => Array.isArray(dest.servers));
+
+        const hasIncorrectMultiSites = !multiSites || multiSites.split(',')
+            .some(site => site.length === 0);
+        const hasValidOptionals = (targetBucket || targetProfile) ?
+            (!(!targetBucket || !targetProfile) && (!hasIncorrectMultiSites &&
```
Comment (philipyoo) [inline bin/replication.js]: if `targetBucket` or `targetProfile` is undefined, using an `&&` here will return an undefined value

**Comment by philipyoo** [comment]
Sorry, will reopen with a jira tag

**Pull request closed without merging**